src/platform/user/profile/vet360/containers/AddressValidationModal.jsx

```diff
@@ -179,13 +185,8 @@ class AddressValidationModal extends React.Component {
       closeModal();
     };
 
-    const confirmedSuggestions = suggestedAddresses.filter(
-      suggestion =>
-        suggestion.addressMetaData?.deliveryPointValidation === CONFIRMED,
-    );
-
     const validationMessageKey = getValidationMessageKey(
-      suggestedAddresses,
+      confirmedSuggestions,
```

Comment:
We need to pass the entire array of `suggestedAddresses` to `getValidationMessageKey()` otherwise the user will never see messages related to bad or missing unit numbers.